src/Utilities/copy.jl

```diff
@@ -22,23 +22,22 @@ function automatic_copy_to(dest::MOI.ModelLike, src::MOI.ModelLike;
 end
 
 """
-    supports_incremental_copy(model::ModelLike, copy_names::Bool)
+    supports_default_copy_to(model::ModelLike, copy_names::Bool)
 
 Return a `Bool` indicating whether the model `model` supports
 [`default_copy_to(model, src, copy_names=copy_names)`](@ref) if all the
 attributes set to `src` and constraints added to `src` are supported by `model`.
 
 This function can be used to determine whether a model can be loaded into
 `model` incrementally or whether it should be cached and copied at once instead.
-This is used by JuMP to determine whether to add a cache or not.
-If this function returns false whatever the value of `copy_names` is then
-JuMP stores a cache in a `CachingOptimizer` handling the names. Then if bridges
-are used, a cache of the bridged model is stored in another `CachingOptimizer`.
-If `supports_incremental_copy(optimizer, false)` is `true` then no cache is
-used by default for the bridged model. If
-`supports_incremental_copy(optimizer, true)` is `true` then no caching is used
-by default to store the model and handling names as names are handled by
-`optimizer`.
+This is used by JuMP to determine whether to add a cache or not in two
+situations:
+1) A first cache can be used to store the model as entered by the user as well
```

Comment:
Is `1)` recognized for lists in markdown? I thought you need to do `1.`.

Reply:
Yes, it also renders as `1.`. I have turned it into `1.` to make it consistent